package.json

```diff
@@ -15,7 +15,7 @@
     "clean": "npm run clean --workspaces --if-present",
     "discovery": "npm-run-all discovery:build discovery:start",
     "discovery:build": "npm run --workspace packages/common --workspace packages/transform build",
-    "discovery:start": "npm run --workspace packages/transform start -- --input-dir `npm prefix`/packages/discovery -o `npm prefix`/packages/common/config",
+    "discovery:start": "npm run --workspace packages/transform start -- --input-dir `npm prefix`/packages/discovery -o `npm prefix`/packages/common/config -p `npm prefix`/public",
```

Comment:
Could we move it to the sitemap creation or prerender? That happens at build time

Reply:
Looking into it :eyeglasses: